src/model.ts

```diff
@@ -124,6 +124,16 @@ export abstract class Model {
     await model.save();
     return model;
   }
+  
+  /**
+   * Remove all records from a table.
+   */
+  public static async truncate(): Promise<void> {
+    // sqlite TRUNCATE is a different command
+    const truncateCommand = this.adapter.type == 'sqlite' ? 'DELETE FROM':'TRUNCATE';
```

Comment:
I think `===` is better at this scenario.

```
const truncateCommand = this.adapter.type === 'sqlite' ? 'DELETE FROM' : 'TRUNCATE';
```

Reply:
I agree, it's safer to check if the types are equal too.